app/controllers/logs_controller.rb

```diff
@@ -84,7 +84,7 @@ def stats
       @transport_per_year[l.name] = [] if @transport_per_year[l.name].nil?
     }
     @transport_per_year.keys.each{ |k|
-      @transport_per_year[k] = @transport_years.collect{ |y| 0 }
+      @transport_per_year[k] = @transport_years.collect{ |_y| 0 }
```

Comment:
Sure! [here](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/UnusedBlockArgument) and [here](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Lint/UnusedMethodArgument).